.github/workflows/neuron-ci.yml

```diff
@@ -235,11 +235,11 @@ jobs:
           touch .nojekyll;
           echo ::set-output name=status::done
           
-#       - name: Deploy 🚀
-#         uses: JamesIves/github-pages-deploy-action@3.7.1
-#         if: steps.documentation.outputs.status == 'done'
-#         with:
-#           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-#           BRANCH: gh-pages # The branch the action should deploy to.
-#           FOLDER: ${{runner.workspace}}/nrn/build/docs # The folder the action should deploy.
-#           CLEAN: false # Automatically remove deleted files from the deploy branch
+      - name: Deploy 🚀
+        uses: JamesIves/github-pages-deploy-action@3.7.1
+        if: steps.documentation.outputs.status == 'done'
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do you need token or permission to set something?

Reply:
It uses existing one and this time it works, so nothing needed (when I PRed from my fork it didn't work). 
Now I just need to publish only if it's a master push, just like travis.